docs/lib/storage/fragments/flutter/getting-started/20_installLib.md

```diff
@@ -5,5 +5,13 @@ dependencies:
   flutter:
     sdk: flutter
   amplify_storage_s3: '<1.0.0'
-}
+  # reminder: amplify_auth_cognito should also be installed
+```
+
+Don't forget to import it in your project:
+
+```dart
+// // main.dart
```

Comment:
Looks like double comments here. (this may be intentional now that I see it also below)

Reply:
intentional